packages/core/core-flows/src/fulfillment/steps/delete-shipping-option-rules.ts

```diff
@@ -33,7 +33,11 @@ export const deleteShippingOptionRulesStep = createStep(
 
     await fulfillmentModule.deleteShippingOptionRules(ids)
 
-    return new StepResponse(null, shippingOptionRules)
+    return new StepResponse({
+      ids,
+      object: "shipping_option_rule",
+      deleted: true,
```

Comment:
thought: think this should just be the IDs ... this shape is very specific to how we want to respond in the HTTP layer

Reply:
I took this shape from other workflows, and it's also the expected response type for deleting the shipping option rule and multiple similar API routes.

I'll update the return types then of these workflows / API routes

Reply:
The return value should have this shape in the API Routes, but not in the workflows

Reply:
Yes I meant to update the API routes for the changes in the workflows